src/config/userSettings.js

```diff
@@ -15,13 +15,15 @@ const userSettings = {
       iconType: iconType.google,
       type: widgetType.number,
       formatter: financeFormatter,
+      heading: 'Rocket Internet Stock', // Only for displaying widget name in settings modal
```

Comment:
This overwrites whatever is set in the configuration. Let's discuss a different solution.